samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
+            * example capture is taken for 64 samples. Each complex sample is of 24 bits.
+            * The capture shown above is just an example. The captured samples will vary from run to run.
+
+
+
+         * To capture AGC adjusted baseband samples after WLAN packet detection, with the following configuration:
+
+           * Channel: 144
+           * Capture length: 64
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_dyn_pkt_cap
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              83E6824F
+              50BCE2D2
+              0A1D63E1
+              1710F0AD
+              0DDFE040
+              25B128F4
+              C51DF1AB
+              B0242261
+              E4EDD719
+              83EE40FF
+              4DD4181F
+              DC31DBFE
+              5BE3E22B
+              D09CE042
+              0CED70E0
+              001050BE
+              2DD4E070
+              24C149F6
+              DF1CF1C2
+              C03A2121
+              E69DC619
+              90EDF0F4
+              2DD0184F
+              DAE1CEFF
+              63E2420C
+              D059E002
+              124DA3DE
+              010B9054
+              9F83000F
+              0FFE3030
+              1703FDBF
+              C0420902
+              D9DE06E0
+              F0D41135
+              9E18DC10
+              0ACFF9FE
+              C82590E0
+              CE0C041D
+              19FF21EA
+              FB0A10E7
+              6EA00C92
+              0FCF7A24
+              2DFCB276
+              5100DA9F
+              FDE1B9D6
+              4E12A177
+              6F64EB52
+              EC1187CD
+
+
+
+         .. note::
+
+            * This test requires a DUT to be connected with VSG using cabled setup. VSG should be continuously transmitting
+            * WLAN packets. Set the desired signal power.
+            * The above capture is taken for 64 samples. Each complex sample is of 24 bits. The console will display in 32 bits
+            * format so there are 48, 32 bit samples.
+            * In the above capture 0xE6824F is first sample, 0xE2D283 is second sample, 0xE150BC is third sample and so on
+            * The capture shown above is just an example. The captured samples will vary from run to run
+            * The above capture is taken after WLAN packet detection. So, this capture will not have first few samples in
+            * first WLAN packet in the capture.
+            * The user is advised to feed smaller packets so that multiple packets can be seen in the capture.
```

Comment:
```suggestion
            * This test requires a DUT to be connected with VSG using cabled setup.
              VSG should be continuously transmitting WLAN packets.
            * Set the desired signal power.
            * The capture is taken for 64 samples.
              Each complex sample is of 24 bits.
              The console will display in 32 bits format so there are 48, 32 bit samples.
            * In the capture, ``0xE6824F`` is the first sample, ``0xE2D283`` the second sample, ``0xE150BC`` the third sample and so on.
            * The capture is just an example.
              The captured samples will vary from run to run
            * The capture is taken after WLAN packet detection, so it will not have the first few samples in the first WLAN packet.
            * Smaller packets should be used so that multiple packets can be seen in the capture.
```